re2.asl

```diff
@@ -477,335 +474,25 @@ split
-                {
-                    if (!vars.Splits.Contains("herbicide"))
-                        return LogAndSplit("herbicide");
-                    break;
-                }
-                case 0x000000B7:
-                {
-                    if (!vars.Splits.Contains("jointPlug"))
-                        return LogAndSplit("jointPlug");
-                    break;
-                }
-                case 0x000000BB:
-                {
-                    if (!vars.Splits.Contains("chipAdmin"))
-                        return LogAndSplit("chipAdmin");
-                    break;
-                }
-                case 0x000000BD:
-                {
-                    if (!vars.Splits.Contains("modulator"))
-                        return LogAndSplit("modulator");
-                    break;
-                }
-                case 0x000000C2:
-                {
-                    if (!vars.Splits.Contains("sewerKey"))
-                        return LogAndSplit("sewerKey");
-                    break;
-                }
-                case 0x000000C3:
-                case 0x000000C8:
-                {
-                    if (!vars.Splits.Contains("generalChipAda") && current.survivorType == 2)
-                        return LogAndSplit("generalChipAda");
-                    if (!vars.Splits.Contains("generalChipMain") && (current.survivorType == 0 || current.survivorType == 1))
-                        return LogAndSplit("generalChipMain");
-                    break;
-                }
-                case 0x000000C4:
-                case 0x000000C9:
-                {
-                    if (!vars.Splits.Contains("staffChip"))
-                        return LogAndSplit("staffChip");
-                    break;
-                }
-                case 0x000000C5:
-                case 0x000000CA:
-                {
-                    if (!vars.Splits.Contains("seniorChip"))
-                        return LogAndSplit("seniorChip");
-                    break;
-                }
-                case 0x000000F0:
-                {
-                    if (!vars.Splits.Contains("fuse1"))
-                        return LogAndSplit("fuse1");
-                    break;
-                }
-                case 0x000000F1:
-                {
-                    if (!vars.Splits.Contains("fuse2"))
-                        return LogAndSplit("fuse2");
-                    break;
-                }
-                case 0x000000F3:
-                {
-                    if (!vars.Splits.Contains("scissors"))
-                        return LogAndSplit("scissors");
-                    break;
-                }
-                case 0x000000F4:
-                    if (!vars.Splits.Contains("boltCutters"))
-                        return LogAndSplit("boltCutters");
-                    break;
-                default:
-                {
-                    break; // No work to do.
-                }

```

Comment:
Removed as part of a refactor - we can cycle through a dict list previously defined, instead of checking these item-by-item